neo/Consensus/ConsensusMessageType.cs

```diff
@@ -10,6 +10,9 @@ internal enum ConsensusMessageType : byte
         PrepareRequest = 0x20,
         [ReflectionCache(typeof(PrepareResponse))]
         PrepareResponse = 0x21,
+        [ReflectionCache(typeof(RecoveryMessage))]
+        RecoveryMessage = 0x22,
```

Comment:
I tend to assign it `0x41`.
If we decide to add `RecoveryRequest` in the future, we can assign it `0x40`.

Reply:
ok, I'll change it.

Reply:
0x00~0x1f: View Changing Protocol
0x20~0x3f: Consensus Protocol
0x40~0x5f: Recovery Protocol